src/SqlView/SqlView.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Paper from 'material-ui/Paper/Paper';
+import RaisedButton from 'material-ui/RaisedButton';
+import FormHeading from '../EditModel/FormHeading';
+import { goToRoute } from '../router-utils';
+import LoadingMask from '../loading-mask/LoadingMask.component';
+import snackActions from '../Snackbar/snack.actions';
+
+const backToList = () => {
+    goToRoute('list/otherSection/sqlView');
+};
+
+class SqlView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            listGrid: null,
+        };
+    }
+
+    async componentWillMount() {
```

Comment:
Async stuff should probably be in componentDidMount. It's also deprecated, so we should try not to use it.

Reply:
Done, see [this commit](https://github.com/dhis2/maintenance-app/pull/481/commits/5d71d2f4fe70679387c3160f1249c21f5a2f0592)